src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+const fs = require('fs')
+const yargs = require('yargs')
+
+const { report_coverage } = require('./report_coverage')
+
+const argv = yargs
+    .usage('Usage: $0 <json_filename> [options]')
+    .command('$0 <json_filename>', '', yargs => {
+        yargs.positional('json_filename', {
+            describe: 'path to coverage-summary.json',
+            type: 'string',
+        })
+    })
+    .example('$0 report.json -fail 90', 'Parses JSON coverage from file, reports to readable python-coverage and checks coverage thresholds.')
+    .option('fail', {
+        alias: 'f',
+        describe:  'coverage fail threshold',
+        default: 0,
+        type: 'number',
+    })
+    .option('regex', {
+        alias: 'r',
+        describe:  'path regex for reported files',
+        default: '',
+        type: 'string',
+    })
+    .help('h')
+    .alias('h', 'help')
+    .argv
+
+if (fs.existsSync(argv.json_filename)){
+    const json_file = JSON.parse(fs.readFileSync(argv.json_filename))
+    report_coverage(json_file, process.stdout, argv.regex, argv.fail)
+}
```

Comment:
In case that the file does not exist, the script should display an error and exit with non-zero status code.

Reply:
Fixed

Reply:
Is it OK to use `console.err`? Shouldn't you use `process.stderr.write` to be consistent with the rest of the code? Also, this branch should be tested.

Reply:
I improved the app as we disccussed before lunch.